### PR TITLE
Terraform011

### PR DESCRIPTION
**Pull request by miguelaferreira**
# Description

This PR uses a vpc module version compatible with terraform 0.11 in examples, and documents how to workaround the issue reported in #18

### CODE REVIEW

File: examples/complete-dual-vpn-gateway/main.tf [hunk 1]
```diff
@@ -47,7 +47,8 @@ resource "aws_customer_gateway" "secondary" {
 }
 
 module "vpc" {
-  source = "git::https://github.com/terraform-aws-modules/terraform-aws-vpc.git"
+  source = "terraform-aws-modules/vpc/aws"
+  version = "1.67.0"
```
Comment (antonbabenko) [inline examples/complete-dual-vpn-gateway/main.tf]: Let's use this instead - `~> 1.0`

File: examples/complete-vpn-gateway-with-static-routes/main.tf [hunk 2]
```diff
@@ -39,6 +39,7 @@ resource "aws_customer_gateway" "main" {
 
 module "vpc" {
   source = "terraform-aws-modules/vpc/aws"
+  version = "1.67.0"
```
Comment (antonbabenko) [inline examples/complete-vpn-gateway-with-static-routes/main.tf]: Same here and in other examples.

File: examples/complete-dual-vpn-gateway/main.tf (hunk 1 above)
Comment (miguelaferreira) [reply examples/complete-dual-vpn-gateway/main.tf]: Done

File: examples/complete-vpn-gateway-with-static-routes/main.tf (hunk 2 above)
Comment (miguelaferreira) [reply examples/complete-vpn-gateway-with-static-routes/main.tf]: Done, for all of them.

**Comment by antonbabenko** [comment]
Great, [v1.7.0](https://github.com/terraform-aws-modules/terraform-aws-vpn-gateway/releases/v1.7.0) has been released.

**Pull request merged**